acceptance/Makefile

```diff
@@ -0,0 +1,183 @@
+PROJECT_NAME ?= clusterman
+WHOAMI := $(shell whoami)
+KIND_CLUSTER ?= $(WHOAMI)-$(PROJECT_NAME)
+KUBECONFIG ?= .local-$(KIND_CLUSTER).conf
+DOCKER_IMAGE ?= $(PROJECT_NAME)-dev-$(WHOAMI)
+DOCKER_PORT ?= 31234
+DOCKER_REGISTRY ?= $(KIND_CLUSTER)-reg
+LOCAL_ENV ?= LOCAL_ENV_$(subst -,_,$(PROJECT_NAME))
+
+.PHONY: local-env-check
+local-env-check:
+	@which kind >/dev/null || (echo 'kind must be installed in PATH' && exit 1)
+	@which kubectl >/dev/null || (echo 'kubectl must be installed in PATH' && exit 1)
+	@test "$$$(LOCAL_ENV)" = "done" || (echo 'Please run `source <(make local-env)`.' && exit 1)
+
+.PHONY: local-env
+local-env:  ## Set env vars in current shell to do local cluster development: source <(make local-env)
+	@echo "export KIND_CLUSTER=$(KIND_CLUSTER)"
+	@echo "export KUBECONFIG=$(KUBECONFIG)"
+	@echo "export $(LOCAL_ENV)=done"
+
+.PHONY: local-cluster-internal
+local-cluster-internal: local-env-check .local-cluster.yaml .local-clusterman-internal.yaml .local-cluster-up
+	@echo Done.
+
+.PHONY: local-cluster-external
+local-cluster-external: local-env-check .local-cluster.yaml .local-clusterman-external.yaml .local-cluster-up
+	@echo Done.
+
+.local-cluster-up:
+	kind create cluster --name $(KIND_CLUSTER) --config .local-cluster.yaml --verbosity 2
+	kubectl -n kube-system get configmap/coredns -o yaml | \
+		sed "s|/etc/resolv.conf|$$(awk '/nameserver/ { print $$2; exit }' /etc/resolv.conf)|" | \
+		kubectl replace -f -
+	./k8s-local-docker-registry.sh $(DOCKER_REGISTRY) $(DOCKER_PORT) $(KIND_CLUSTER)
+	docker tag $(DOCKER_IMAGE) localhost:$(DOCKER_PORT)/$(DOCKER_IMAGE)
+	docker push localhost:$(DOCKER_PORT)/$(DOCKER_IMAGE)
+	docker exec $(KIND_CLUSTER)-control-plane mkdir -p /var/lib/clusterman
+	docker cp $(KUBECONFIG) $(KIND_CLUSTER)-control-plane:/var/lib/clusterman/clusterman.conf
+	docker exec $(KIND_CLUSTER)-control-plane chmod 644 /var/lib/clusterman/clusterman.conf
+	docker exec $(KIND_CLUSTER)-control-plane sed -i "s/\(.*server: \).*/\1https:\\/\\/127.0.0.1:6443/g" /var/lib/clusterman/clusterman.conf
+	COMPOSE_PROJECT_NAME=$(PROJECT_NAME)_k8s ../.tox/acceptance/bin/docker-compose -f docker-compose-k8s.yaml up -d moto-ec2 moto-s3 moto-dynamodb
+	sleep 10
+	docker ps
+	ACCEPTANCE_ROOT=. DISTRIB_CODENAME=bionic python run_instance.py \
+		$$(docker ps | grep clusterman_k8s.*ec2 2>&1 | sed -e 's|^.*:\(.*\)->5000.*$$|http://127.0.0.1:\1|') \
+		$$(docker ps | grep clusterman_k8s.*s3 2>&1 | sed -e 's|^.*:\(.*\)->5000.*$$|http://127.0.0.1:\1|') \
+		$$(docker ps | grep clusterman_k8s.*dynamodb 2>&1 | sed -e 's|^.*:\(.*\)->5000.*$$|http://127.0.0.1:\1|') \
```

Comment:
should probably be using `$(PROJECT_NAME)_k8s`, not `clusterman_k8s` here.

Is there no easy way to get this from `docker-compose`? Maybe parse `docker-compose ps` instead of `docker ps`?